scripts/make_wheel.sh

```diff
@@ -120,7 +120,7 @@ build_source() {
 
   if [[ $(($python27 + $python35 + $python36)) -gt 1 ]]; then
     echo "Two or more versions of Python specified. Pick one."
-    exit 1
+    return 1
```

Comment:
Will this change the behavior of the script? I think it may have been intended to early-exit when this error occurs.